mod_cluster/extension/src/main/resources/org/wildfly/extension/mod_cluster/LocalDescriptions.properties

```diff
@@ -50,8 +50,9 @@ modcluster.load-provider.simple.remove=Remove a simple load provider.
 modcluster.load-provider.simple.factor=Dynamic load provider computes the load balance factor of a node from a defined set of load metrics.
 modcluster.load-provider.dynamic.add=Add a dynamic load provider with no load metrics.
 modcluster.load-provider.dynamic.remove=Remove a dynamic load provider defaulting to simple load provider.
-modcluster.load-provider.dynamic.history=The number of historic (previous) load values to consider in the load balance factor computation.
 modcluster.load-provider.dynamic.decay=The factor by which a historic load values should degrade in significance.
+modcluster.load-provider.dynamic.history=The number of historic (previous) load values to consider in the load balance factor computation.
+modcluster.load-provider.dynamic.initial-load=Initial load within the range [0..100] with which to prepopulate historical values. Used to gradually drive load to the node. Value of 0 prepopulates with full load and value of -1 disables this behavior.
```

Comment:
Wouldn't a value of -1 put the worker in "hot standby"?  A value of 100 would effectively disable this behavior, no?

Reply:
This comment is not in response to Paul.

This needs some good docs. :)  AIUI this is really more like 'desired-initial-load', but the already released mod-cluster config property is initial-load, so it's likely this attribute has to be named the same in order to avoid confusion.

AIUI, what the user is setting here is how much load they *want* the worker to receive initially, moving over time to the fully dynamic value as history/decay kicks in.  So '0' means, 'I'm fully loaded, give me nothing'. It is not how much load the worker should report, e.g. it is not an 'initial-load-factor', i.e. the load factor value the node directly reports, moving over time to the fully dynamic value as history/decay kicks in.  TBH when I see this my intuitive understanding is it means 'initial-load-factor' which is the opposite of reality.

Reply:
@pferraro  Actually, previous versions do not do any sort of pre-population like this, so the legacy value is really "-1", i.e. disable the behavior.

Reply:
@bstansberry This factor stuff is very confusing for everyone to this day. We just need to rely on docs here..

Reply:
Where are those docs?  https://docs.modcluster.io/#_worker_side_load_metrics doesn't include this property, although AIUI the property is available in mod_cluster.

Reply:
Added the description of the attribute now in the docs. We were (and still kinda are) in the process of moving to the new docs site that is https://docs.modcluster.io/